spec/requests/objects_controller_content_diff_spec.rb

```diff
@@ -64,8 +64,9 @@
           params = { content_metadata: content_md, subset: 'unrecognized' }
           post content_diff_object_url(id: prefixed_druid), params: params, headers: valid_auth_header
           expect(response).to have_http_status(:bad_request)
-          m = "400 Bad Request: subset arg must be 'all', 'shelve', 'preserve', or 'publish' (MoabStorageService.content_diff for druid bj102hs9687)"
-          expect(response.body).to eq m
+          error_response = JSON.parse(response.body)['errors'].first
+          expect(error_response['status']).to eq('bad_request')
+          expect(error_response['detail']).to include('unrecognized isn\'t include enum') # sic
```

Comment:
ew - yucky error message.  Committee gem giveth and taketh away.

Reply:
yeah :unamused: